custom_components/opnsense/device_tracker.py

```
@@ -35,257 +35,196 @@
+    # use configured mac addresses if setup, otherwise create an entity per arp entry
+    arp_entries: list = dict_get(state, "arp_table")
+    if not isinstance(arp_entries, list):
+        arp_entries = []
+    devices: list = []
+    mac_addresses = []
+    configured_mac_addresses = config_entry.options.get(CONF_DEVICES, [])
+    if configured_mac_addresses and config_entry.options.get(
+        CONF_DEVICE_TRACKER_ENABLED, DEFAULT_DEVICE_TRACKER_ENABLED
+    ):
+        _LOGGER.debug(
+            f"[device_tracker async_setup_entry] configured_mac_addresses: {configured_mac_addresses}"
```

Ruff

Logging statement uses f-string

(G004)